app/pages/cif/project-revision/[projectRevision]/form/[formIndex]/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable relay/must-colocate-fragment-spreads*/
```

Comment:
OK to this globally for this file? Otherwise I'd have to do it 15 times for all the fragments

Reply:
I think makes sense for that file

Reply:
@BCerki Not sure but I think we need to be more specific here regarding rules. according to this [page](https://github.com/relayjs/eslint-plugin-relay/blob/main/eslint-plugin-relay.js), some rules under strict mode don't comply with our project (like `flow types`). But maybe I'm wrong 😃 worth asking more experienced devs.